phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1451,7 +1454,35 @@ private TableDescriptor ensureTableCreated(byte[] physicalTableName, PTableType
         }
         return null; // will never make it here
     }
-    
+
+    private boolean isSplitPolicyUpdatedForTaskTable(
+            final TableDescriptorBuilder tdBuilder) {
```

Comment:
Can we add a new unit test for this method?

Reply:
Done